### PR TITLE
tiup: restriction about DXF

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: upgrade-tidb-using-tiup.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ tiup update cluster
 
 - 集群 DDL 情况：
 
-    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要检查集群的 DDL 情况，无需等待 DDL job 完成或取消正在进行的 DDL job。
+    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要检查集群的 DDL 情况，无需等待 DDL job 完成或取消正在进行的 DDL job。但如果是升级到 v8.1.0 或之后的版本且开启了[分布式执行框架](/tidb-distributed-execution-framework)，建议关闭分布式执行框架后，再升级，否则升级期间添加的索引可能会存在和数据不一致，升级失败的问题。
```
Comment (hfxsd) [inline upgrade-tidb-using-tiup.md:177]:
```suggestion
    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要检查集群的 DDL 情况，无需等待 DDL job 完成或取消正在进行的 DDL job。但如果是升级到 v8.1.0 或更高版本，且开启了[分布式执行框架](/tidb-distributed-execution-framework.md)，建议先关闭分布式执行框架后，然后再升级，否则升级期间添加的索引可能会出现和数据不一致的问题，导致升级失败。 
```

File: upgrade-tidb-using-tiup.md [hunk 2]
```diff
@@ -174,7 +174,7 @@ tiup update cluster
 
 - 集群 DDL 情况：
 
-    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要检查集群的 DDL 情况，无需等待 DDL job 完成或取消正在进行的 DDL job。
+    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要检查集群的 DDL 情况，无需等待 DDL job 完成或取消正在进行的 DDL job。但如果是升级到 v8.1.0 或更高版本，且开启了[分布式执行框架](/tidb-distributed-execution-framework.md)，建议先关闭分布式执行框架后，然后再升级，否则升级期间添加的索引可能会出现和数据不一致的问题，导致升级失败。
```
Comment (hfxsd) [inline upgrade-tidb-using-tiup.md:177]:
```suggestion
    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要检查集群的 DDL 情况，无需等待 DDL job 完成或取消正在进行的 DDL job。但如果是升级到 v8.1.0 或更高版本，且开启了[分布式执行框架](/tidb-distributed-execution-framework.md)，建议关闭分布式执行框架后再升级，否则升级期间添加的索引可能会出现和数据不一致的问题，导致升级失败。 
```

File: upgrade-tidb-using-tiup.md [hunk 3]
```diff
@@ -174,7 +174,7 @@ tiup update cluster
 
 - 集群 DDL 情况：
 
-    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要检查集群的 DDL 情况，无需等待 DDL job 完成或取消正在进行的 DDL job。
+    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要检查集群的 DDL 情况，无需等待 DDL job 完成或取消正在进行的 DDL job。但如果是升级到 v8.1.0 或更高版本，且开启了[分布式执行框架](/tidb-distributed-execution-framework.md)，建议关闭分布式执行框架后再升级，否则升级期间添加的索引可能会出现和数据不一致的问题，导致升级失败。
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:177]:
```suggestion
    - 如果你采用[平滑升级](/smooth-upgrade-tidb.md) 到 v8.1.0 或更高版本，且开启了[分布式执行框架](/tidb-distributed-execution-framework.md)，建议关闭分布式执行框架后再升级，否则升级期间添加的索引可能会出现和数据不一致的问题，导致升级失败。 
```

Comment (Frank945946) [reply upgrade-tidb-using-tiup.md:177]: 对的。未来升级到 v8.1 或者 v8.5 的 patch 版本会解除这个限制

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**